tgui/packages/tgui/interfaces/IntegratedCircuit/DisplayName.js

```diff
@@ -10,8 +10,8 @@ export const DisplayName = (props, context) => {
   const InputComponent = FUNDAMENTAL_DATA_TYPES[port.type || 'unknown'];
   const TypeDisplayHandler = DATATYPE_DISPLAY_HANDLERS[port.type || 'unknown'];
 
-  const hasInput
-    = !isOutput && port.connected_to === NULL_REF && InputComponent;
+  const hasInput = !isOutput
+    && InputComponent;
```

Comment:
```suggestion
    port.connected_to === NULL_REF && InputComponent;
```

Reply:
`connected_to` would be an empty list, not a `NULL_REF`.

Reply:
fixed